docs/getting_started/index.rst

```diff
@@ -144,42 +186,50 @@ try running some command line tools:
 .. code-block:: console
 
     $ ctapipe-info --all
-    $ ctapipe-camdemo --camera=NectarCam  # try --help for more info
+    $ ctapipe-process -i dataset://gamma_prod5.simtel.zst -o test.h5  # try --help for more info
 
 To update to the latest development version (merging in remote changes
 to your local working copy):
 
 .. code-block:: console
 
-   $ git pull upstream master
+   $ git fetch upstream
+   $ git merge upstream/main # or rebase, see above
 
 ---------------------------------------
 Developing a new feature or code change
 ---------------------------------------
 
-We adhere to the PEP8 coding style (see our :doc:`/development/style-guide`).
-To enforce this, setup the
+We are using the ``black`` and ``isort`` auto-formatters for automatic 
+adherence to the code style (see our :doc:`/development/style-guide`).
+To enforce running these tools whenever you make a commit, setup the
 `pre-commit hook <https://pre-commit.com/>`_::
 
     $ pre-commit install
```

Comment:
Suggested addition about how the pre-commit hook will block your commit. Something like:

> The pre-commit hook will then execute the tools with the same settings as when the a pull request is checked on github, and if any problems are reported the commit will be rejected. You then have to fix the reported issues before tying to commit again. 
> Note that a common problem is forgetting to run the `black` code formatter, and that whenever this was the only problem found, simply adding the changes resulting from the pre-commit hook to the commit will result in your changes being accepted.